### PR TITLE
Remove duplicated code with a CliCommandInfo constructor

### PR DESCRIPTION
**Pull request by sakridge**
#### Problem

Setting no signers is the most common case to construct a CliCommandInfo struct, 

#### Summary of Changes

Make a constructor function to do that and reduce code.

Fixes #

### CODE REVIEW

File: cli/src/cli.rs [hunk 1]
```diff
@@ -461,6 +461,15 @@ pub struct CliCommandInfo {
     pub signers: CliSigners,
 }
 
+impl CliCommandInfo {
+    pub fn new(command: CliCommand) -> Self {
```
Comment (CriesofCarrots) [inline cli/src/cli.rs]:
```suggestion
    pub fn new_unsigned(command: CliCommand) -> Self {
```
I'm not totally set on this name, but something that makes it clear that signers will be populated as empty would be nice.

Comment (CriesofCarrots) [reply cli/src/cli.rs]: Could have a constructor that takes signers as well.

**Review by CriesofCarrots — COMMENTED** [review]
Generally on board. This is still marked as draft, though...?

File: cli/src/cli.rs (hunk 1 above)
Comment (joncinque) [reply cli/src/cli.rs]: For what it's worth, I kind of like `new_unsigned` even though it isn't perfect. I came up with `new_without_signers` or just `without_signers`, a bit like `Vec::with_capacity`.

Comment (sakridge) [reply cli/src/cli.rs]: I kind of like the `without_signers` name

Comment (CriesofCarrots) [reply cli/src/cli.rs]: `without_signers` is fine

Comment (sakridge) [reply cli/src/cli.rs]: ok. updated to that

**Review by joncinque — APPROVED** [review]

**Review by CriesofCarrots — APPROVED** [review]

**Pull request merged**